src/extract.ts

```diff
@@ -133,6 +133,39 @@ export interface PytestBenchmarkJson {
     version: string;
 }
 
+type JuliaBenchmarkTrialEstimate = [
+    string,
+    {
+        params: [
+            string,
+            {
+                seconds: number;
+                samples: number;
+                evals: number;
+                overhead: number;
+                gctrial: boolean;
+                gcsample: boolean;
+                time_tolerance: number;
+                memory_tolerance: number;
+            },
+        ];
+        time: number;
+        gctime: number;
+        memory: number;
+        allocs: number;
+    },
+];
```

Comment:
looking at the output I think this can be
```suggestion
type JuliaBenchmarkTrialEstimate = [
    'TrialEstimate',
    {
        params: [
            string,
            {
                seconds: number;
                samples: number;
                evals: number;
                overhead: number;
                gctrial: boolean;
                gcsample: boolean;
                time_tolerance: number;
                memory_tolerance: number;
            },
        ];
        time: number;
        gctime: number;
        memory: number;
        allocs: number;
    },
];
```